media/magicavoxel_vox.ksy

```diff
@@ -0,0 +1,187 @@
+meta:
+  id: vox
+  file-extension: vox
+  application: MagicaVoxel
+  endian: le
+  license: MIT
+  title: MagicaVoxel File
+doc-ref: 'https://ephtracy.github.io/ MagicaVoxel Homepage'
+doc-ref: 'https://github.com/ephtracy/voxel-model/blob/master/MagicaVoxel-file-format-vox.txt Format Description'
+seq:
+  - id: magic
+    contents: 'VOX '
+  - id: version
+    type: u4
+    doc: 150 expected
+  - id: main
+    type: chunk
+types:
+  chunk:
+    seq:
+      - id: chunk_id
+        type: u4be
+        enum: chunk_type
+      - id: num_bytes_of_chunk_content
+        type: u4
```

Comment:
Customarily, we advice to use `len_` prefix for attributes that designate number of bytes (i.e. length or size) of some value — see [style guide](http://doc.kaitai.io/ksy_style_guide.html#attr-id).